tests/webhook/test_delete_webhook.rs

```diff
@@ -0,0 +1,71 @@
+use helius::config::Config;
+use helius::rpc_client::RpcClient;
+use helius::types::{Cluster, HeliusEndpoints};
+use helius::Helius;
+use mockito::Server;
+use reqwest::Client;
+use std::sync::Arc;
+
+#[tokio::test]
+async fn test_delete_webhook_success() {
+    let mut server: Server = Server::new_with_opts_async(mockito::ServerOpts::default()).await;
+    let url: String = format!("{}/", server.url());
+
+    server
+        .mock("DELETE", "/v0/webhooks/0e8250a1-ceec-4757-ad69/?api-key=fake_api_key")
+        .with_status(200)
+        .with_header("Content-Type", "application/json")
+        .create();
+
+    let config: Arc<Config> = Arc::new(Config {
+        api_key: "fake_api_key".to_string(),
+        cluster: Cluster::Devnet,
+        endpoints: HeliusEndpoints {
+            api: url.to_string(),
+            rpc: url.to_string(),
+        },
+    });
+
+    let client: Client = Client::new();
+    let rpc_client: Arc<RpcClient> = Arc::new(RpcClient::new(Arc::new(client.clone()), Arc::clone(&config)).unwrap());
+    let helius = Helius {
+        config,
+        client,
+        rpc_client,
+    };
+
+    let response = helius.delete_webhook("0e8250a1-ceec-4757-ad69").await;
+    //assert!(response.is_ok(), "The API call failed: {:?}", response.err());
```

Comment:
I've merged #28 to fix how we handle empty response bodies so we can test deleting webhooks no problem now